game/src/main.rs

```diff
@@ -11,5 +13,8 @@ fn main() {
         .add_plugins(PhysicsPlugins::default())
         .add_plugins(LdtkPlugin)
         .add_plugins(CharacterControllerPlugin)
+        //user plugins
+        //debug plugin
```

Comment:
You can remove the "debug plugin" comment. The code is self explanatory.

Reply:
Updated